internal/config/parser/known.go

```diff
@@ -0,0 +1,70 @@
+package parser
+
+// List of known environment variables. The first column represents
+// internal config (YAML) version in uppercase-dot notation of the configuration
+// library.
+//
+// The second column is optional. When not set, environment variable is the same,
+// except "." are replaced with "_". Otherwise the value from the second column
+// is used. Examples:
+//
+// APP_NAME (implicit from APP.NAME)
+// AWS_INSTANCE_PREFIX (explicitly set)
+//
+var known = [][]string{
+	{"APP", ""},
+	{"APP.NAME", ""},
+	{"APP.PORT", ""},
+	{"APP.VERSION", ""},
+	{"AWS", ""},
+	{"AWS.INSTANCEPREFIX", "AWS_INSTANCE_PREFIX"},
+	{"AWS.KEY", ""},
+	{"AWS.REGION", ""},
+	{"AWS.SECRET", ""},
+	{"AWS.SESSION", ""},
+	{"CLOUDWATCH", ""},
+	{"CLOUDWATCH.ENABLED", ""},
+	{"CLOUDWATCH.GROUP", ""},
+	{"CLOUDWATCH.KEY", ""},
+	{"CLOUDWATCH.REGION", ""},
+	{"CLOUDWATCH.SECRET", ""},
+	{"CLOUDWATCH.SESSION", ""},
+	{"CLOUDWATCH.STREAM", ""},
+	{"DATABASE", "DATABASE"},
+	{"DATABASE.HOST", ""},
+	{"DATABASE.LOGLEVEL", "DATABASE_LOG_LEVEL"},
+	{"DATABASE.NAME", ""},
+	{"DATABASE.PASSWORD", ""},
+	{"DATABASE.PORT", ""},
+	{"DATABASE.SEEDSCRIPT", "DATABASE_SEED_SCRIPT"},
+	{"DATABASE.USER", ""},
+	{"FEATUREFLAGS", "FEATURE_FLAGS"},
+	{"FEATUREFLAGS.ENVIRONMENT", "FEATURE_FLAGS_ENVIRONMENT"},
+	{"FEATUREFLAGS.EXITONPANIC", "FEATURE_FLAGS_EXIT_ON_PANIC"},
+	{"LOGGING", ""},
+	{"LOGGING.LEVEL", ""},
+	{"LOGGING.MAXFIELD", "LOGGING_MAX_FIELD"},
+	{"LOGGING.STDOUT", ""},
+	{"PROMETHEUS", ""},
+	{"PROMETHEUS.PATH", ""},
+	{"PROMETHEUS.PORT", ""},
+	{"RESTENDPOINTS", "REST"},
+	{"RESTENDPOINTS.IMAGEBUILDER", "REST_IMAGEBUILDER"},
+	{"RESTENDPOINTS.IMAGEBUILDER.PASSWORD", "REST_IMAGEBUILDER_PASSWORD"},
+	{"RESTENDPOINTS.IMAGEBUILDER.PROXY", "REST_IMAGEBUILDER_PROXY"},
+	{"RESTENDPOINTS.IMAGEBUILDER.PROXY.URL", "REST_IMAGEBUILDER_PROXY_URL"},
+	{"RESTENDPOINTS.IMAGEBUILDER.URL", "REST_IMAGEBUILDER_URL"},
+	{"RESTENDPOINTS.IMAGEBUILDER.USERNAME", "REST_IMAGEBUILDER_USERNAME"},
+	{"RESTENDPOINTS.SOURCES", "REST_SOURCES"},
+	{"RESTENDPOINTS.SOURCES.URL", "REST_SOURCES_URL"},
+	{"RESTENDPOINTS.SOURCES.USERNAME", "REST_SOURCES_USERNAME"},
+	{"RESTENDPOINTS.SOURCES.PASSWORD", "REST_SOURCES_PASSWORD"},
+	{"RESTENDPOINTS.SOURCES.PROXY", "REST_SOURCES_PROXY"},
+	{"RESTENDPOINTS.SOURCES.PROXY.URL", "REST_SOURCES_PROXY_URL"},
+	{"RESTENDPOINTS.TRACEDATA", "REST_TRACE_DATA"},
+	{"WORKER", ""},
+	{"WORKER.CONCURRENCY", ""},
+	{"WORKER.HEARTBEATSEC", "WORKER_HEARTBEAT_SEC"},
+	{"WORKER.MAXBEATS", "WORKER_MAX_BEATS"},
+	{"WORKER.QUEUE", ""},
+}
```

Comment:
This file needs to be updated when we add a new configuration value, if not, pbapi will issue a warning in logs naming exactly what is missing here.